### ISSUE TITLE
How can I check if the input-field is currently focused to set class on field-wrapper

### ISSUE THREAD

**Issue by FloNeu** [ngx-formly/ngx-formly#3457]
I had to write a wrapper displaying some additional-borders and styles around an formly form-field.
Now i should update that outer borders with an outline when the 'native' form-field element receives
focus.

I have tried to get somewhere with template-options - and thought i found some solutions, but that was and old angular-formly(unrelated/archived project) question?

What would be the recommended approach to do this?

**Comment by FloNeu** [comment]
Ok... what i found to be true is that the solution i found for angular-formly was still somehow valid.
I finally got it to work like so: FieldConfig: 
`
{
...
    props: {
     ...
      hasFocus: false,
      focus: function( field ) {
        console.log('CALLED_BLUR');
        if( field.props ) {
          field.props['hasFocus'] = true
        }
      },
      blur: function( field ) {
        console.log('CALLED_BLUR');
        if( field.props ) {
          field.props['hasFocus'] = true
        }
      },
    }
...
}
`
So basically i use the blur and focus 'Event' to set my hasFocus template property... Thing that was confusing and made me search for this forever was that i expected these 'blur' 'focus' events on the expression-properties or similar as i haven't had events on the props object before.

Thx anyway. Great project
